contracts/SeriesIndexer.sol

```diff
@@ -0,0 +1,448 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol" as Ownable;
+
+contract SeriesIndexer is Ownable.Ownable {
+    // Counter
+    uint256 private nextSeriesID = 1;
+    uint256 private nextArtistID = 1;
+
+    struct Series {
+        string metadata;
+        string contractTokenData;
```

Comment:
Same as above [comment](https://github.com/bitmark-inc/feralfile-exhibition-smart-contract/pull/44#discussion_r1895217854), it should be `tokenDataURI`